core/src/main/java/com/linecorp/armeria/client/ClientFactoryOptions.java

```diff
@@ -241,6 +241,13 @@ private static long clampedDefaultMaxClientConnectionAge() {
     public static final ClientFactoryOption<ConnectionPoolListener> CONNECTION_POOL_LISTENER =
             ClientFactoryOption.define("CONNECTION_POOL_LISTENER", ConnectionPoolListener.noop());
 
+    /**
+     * The listener which is notified on a connection pool event.
+     */
+    public static final ClientFactoryOption<ClientConnectionEventListener> CLIENT_CONNECTION_EVENT_LISTENER =
```

Comment:
We don't add `CLIENT` because it's already a client option.
```suggestion
    public static final ClientFactoryOption<ClientConnectionEventListener> CONNECTION_EVENT_LISTENER =
```